### PR TITLE
update url after repo layout change

### PR DESCRIPTION
**Pull request by Skgland**

### CODE REVIEW

File: 2021/scripts.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ window.addEventListener("load", async () => {
       lang: _ => "rust",
       solutionUrl: (day, part) => {
         const _day = (day + 1).toString().padStart(2, "0");
-        return `https://raw.githubusercontent.com/Skgland/Advent-of-Code-2021/main/year2021/src/day${_day}.rs`
+        return `https://raw.githubusercontent.com/Skgland/Advent-of-Code-2021/main/crates/year2021/src/day${_day}.rs`
```
Comment (melfkammholz) [inline 2021/scripts.js:118]: You forgot to change the year in your repository name.

Comment (Skgland) [reply 2021/scripts.js:118]: yeah, looks like I never updated it when I removed the year from the year from the repo name

Comment (Skgland) [inline 2021/scripts.js:118]:
```suggestion
        return `https://raw.githubusercontent.com/Skgland/Advent-of-Code/main/crates/year2021/src/day${_day}.rs`
```

Comment (Skgland) [reply 2021/scripts.js]: fixed

**Pull request merged**